test/endpoint_tests/conftest.py

```diff
@@ -0,0 +1,39 @@
+import os
+import tempfile
+
+import pytest
+
+from boxwise_flask.app import create_app
+from boxwise_flask.db import db
+from boxwise_flask.models import Camps, Cms_Usergroups_Camps, Cms_Users, Person
+
+MODELS = (Person, Camps, Cms_Usergroups_Camps, Cms_Users)
+
+
+@pytest.fixture()
+def app():
+    app = create_app()
+
+    db_fd, db_filepath = tempfile.mkstemp(suffix=".sqlite3")
```

Comment:
Out of curiosity: Why did you not go with an in-memory DB (like in different places or tests IIRC)

Reply:
When you close the connection it deletes it all